app/api/services/supplier_domain.py

```diff
@@ -47,3 +47,13 @@ def set_supplier_domain_status(self, supplier_id, domain_id, status, price_statu
                 price_status=price_status
             )
             return self.save(supplier_domain, do_commit)
+
+    def get_supplier_domain_by_id_and_suppler_id(self, id, supplier_id):
```

Comment:
```suggestion
    def get_supplier_domain_by_id_and_supplier_id(self, id, supplier_id):
```
You might need to add this change to other parts of your code